shared/Microsoft.Extensions.WebEncoders.Sources/WebEncoders.cs

```diff
@@ -3,8 +3,15 @@
 
 using System;
 using System.Diagnostics;
-using System.Globalization;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
 using Microsoft.Extensions.WebEncoders.Sources;
+using System.Buffers;
```

Comment:
nit: sort usings

Reply:
Done -- will be updated with the other feedback changes.